clang/lib/Format/ContinuationIndenter.cpp

```diff
@@ -1334,12 +1334,15 @@ unsigned ContinuationIndenter::getNewLineColumn(const LineState &State) {
            Style.IndentWidth;
   }
 
-  if (Style.BraceWrapping.BeforeLambdaBody &&
-      Style.BraceWrapping.IndentBraces && Current.is(TT_LambdaLBrace)) {
+  if (Style.BraceWrapping.BeforeLambdaBody && Current.is(TT_LambdaLBrace)) {
+    const auto Nested = Current.NestingLevel != 0;
```

Comment:
```suggestion
    const auto Nested = Current.NestingLevel > 0;
```
Should be the same, but can't let people wonder if `NestingLevel` can be negative.

Reply:
Good idea.